neqo-http3/src/lib.rs

```diff
@@ -99,6 +102,24 @@ impl Error {
             _ => 3,
         }
     }
+
+    // TODO: dragana look into http3 errors
```

Comment:
Can you be more specific?

Reply:
I am looking into this #757 . I will write down which error can happen in which function and how and where to handle it (my last 2 patches are made while I was looking into this and some more are coming)